rtic-monotonics/src/systick.rs

```diff
@@ -5,7 +5,7 @@
 //!
 //! | Feature          | Tick rate | Precision |
 //! |:----------------:|----------:|----------:|
-//! | (none / default) |  1 Hz     |      1 ms |
+//! | (none / default) |  1 Hz     |       1 s |
```

Comment:
This should say 1 kHz, 1 ms.

Reply:
My bad, I corrected it in the wrong direction.